build.sbt

```diff
@@ -20,14 +21,17 @@ inThisBuild(
       ),
     ),
     testFrameworks += new TestFramework("munit.Framework"),
+    scalacOptions -= "-Xlint:byname-implicit",
```

Comment:
byname implicits are implicits expanded lazily (see `Lazy` in shapeless) which are needed to work around the default mechanism of implicit resolution. More info here https://docs.scala-lang.org/sips/byname-implicits.html

Now, this line disables `-Xlint` for byname-implicit but I don't remember why I have added it (it was a long time ago). My best guess is that in 2.13 it would trigger spurious errors, but I'll take a closer look